commands/update.js

```diff
@@ -0,0 +1,58 @@
+const exec = require('child_process').exec;
+
+module.exports = {
+  describe: 'Update the SparkPost CLI',
+  action: function(callback) {
+    let packageName = require('../package.json').name;
+
+    isFromNPM(packageName, function(fromNPM) {
+      if (!fromNPM) {
+        return callback(new Error('Cannot update CLI via NPM.\nIf you cloned it from github pull the latest version to update.'));
+      }
+      else {
+        update(packageName, callback);
+      }
+    });
+  }
+};
+
+function isFromNPM(packageName, callback) {
+  exec('npm list -g --depth=0 2>/dev/null', function(err, stdout, stderr) {
+    // something went wrong (no npm, no global, idk)
+    if (err || stderr) {
```

Comment:
Given a standard `(err, result)` callback, this would be an error case rather than a "package is not from NPM" result.